### PR TITLE
Fixed some bugs encountered while EKS Cluser Create Testing

### PR DESCRIPTION
**Pull request by patil-pratik-87**

### CODE REVIEW

File: src/app/core/components/wizard/Wizard.js [hunk 1]
```diff
@@ -208,6 +208,15 @@ class Wizard extends PureComponent {
       )
       .flat()
       .filter(Boolean)
+
```
Comment (kevinylee1991) [inline src/app/core/components/wizard/Wizard.js:211]: What bug is this fixing? Am a bit confused where this is necessary

File: src/app/plugins/infrastructure/components/clusters/aws/capi/capi-vpc/CapiPmkManagedSubnets.tsx [hunk 2]
```diff
@@ -75,11 +97,12 @@ export default function CapiPmkManagedSubnets({ wizardContext, setWizardContext,
   const [privateSubnetsCount, setPrivateSubnetsCount] = useState(1)
   const [publicSubnetsCount, setPublicSubnetsCount] = useState(1)
 
-  const [subnets, dispatch] = useReducer<PmkManagedSubnetsReducer>(subnetsReducer, [
-    getDefaultSubnetCidrBlock(true, az),
-    getDefaultSubnetCidrBlock(false, az),
-  ])
+  const initalState = useMemo(
```
Comment (kevinylee1991) [inline src/app/plugins/infrastructure/components/clusters/aws/capi/capi-vpc/CapiPmkManagedSubnets.tsx:100]: initialState typo

**Review by kevinylee1991 — COMMENTED** [review]
Changes seem fine to me

File: src/app/core/components/wizard/Wizard.js (hunk 1 above)
Comment (patil-pratik-87) [reply src/app/core/components/wizard/Wizard.js:211]: Did you mean the `filter(Boolean)` or the duplicate function?

File: src/app/plugins/infrastructure/components/clusters/aws/capi/capi-vpc/CapiPmkManagedSubnets.tsx (hunk 2 above)
Comment (patil-pratik-87) [reply src/app/plugins/infrastructure/components/clusters/aws/capi/capi-vpc/CapiPmkManagedSubnets.tsx:100]: thanks

File: src/app/core/components/wizard/Wizard.js (hunk 1 above)
Comment (patil-pratik-87) [reply src/app/core/components/wizard/Wizard.js:211]: We are removing duplicate parsed schemas. For instance, in EKS the Infra and control plane screens both need to show AWSManagedControlPlane which leads to duplicates but essentially while updating we need only one.

**Review by kevinylee1991 — APPROVED** [review]

**Pull request merged**